src/Events.Functions/Clients/OutboundClient.cs

```diff
@@ -6,54 +6,54 @@
 using System.Text.Json;
 using System.Threading.Tasks;
 using Altinn.Common.AccessTokenClient.Services;
+using Altinn.Platform.Events.Functions.Clients.Interfaces;
 using Altinn.Platform.Events.Functions.Configuration;
 using Altinn.Platform.Events.Functions.Extensions;
 using Altinn.Platform.Events.Functions.Models;
 using Altinn.Platform.Events.Functions.Services.Interfaces;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 
-namespace Altinn.Platform.Events.Functions.Services
+namespace Altinn.Platform.Events.Functions.Clients
 {
     /// <summary>
     /// Handles PushEvents service
     /// </summary>
-    public class PushEventsService : IPushEventsService
+    public class OutboundClient : IOutboundClient
```

Comment:
I would suggest limiting this PR to changes in the events app, and leaving the function changes for the issue related to the Azure Function.